app/ide-desktop/lib/dashboard/src/modals/UpsertSecretModal.tsx

```diff
@@ -80,16 +85,26 @@ export default function UpsertSecretModal(props: UpsertSecretModalProps) {
         </label>
         <label className="relative flex h-row items-center">
           <div className="text w-modal-label">{getText('value')}</div>
-          <input
-            autoFocus={!isNameEditable}
-            placeholder={
-              isNameEditable ? getText('secretValuePlaceholder') : getText('secretValueHidden')
-            }
-            className="text grow rounded-full border border-primary/10 bg-transparent px-input-x"
-            onInput={event => {
-              setValue(event.currentTarget.value)
-            }}
-          />
+          <div className="relative grow">
+            <input
```

Comment:
Hope eventually it'd a separate `<Password />` component

Reply:
yeah definitely. the issue is that we have multiple *slightly* different styles for the various areas (auth, settings, modals) and so we should *probably* figure out how to unify them first (or i suppose we can make the component styleable)

Reply:
It's technically possible using [`tw-merge`](https://www.npmjs.com/package/tailwind-merge) - we can override default styles in place, and tw-merge will dedupe classes